docs/30_spectra/10_organic_chemistry/chromatography/introduction/README.md

```diff
@@ -0,0 +1,11 @@
+# GC/LC-MS chromatography 
+
+https://en.wikipedia.org/wiki/Gas_chromatography
+
+https://en.wikipedia.org/wiki/Liquid_chromatography%E2%80%93mass_spectrometry
```

Comment:
it is better to use proper links, ideally in the text, e.g., 

`Chromatography, such as [gas chromatography](https://en.wikipedia.org/wiki/Gas_chromatography) or [liquid chromatography](https://en.wikipedia.org/wiki/Liquid_chromatography%E2%80%93mass_spectrometry), ...`